packages/core/src/routes/session.test.ts

```diff
@@ -149,22 +147,40 @@ afterEach(() => {
   interactionResult.mockClear();
 });
 
+const mockInteraction = {
+  jti: 'sessionId',
+  params: { client_id: 'clientId' },
+};
+const getInteraction = jest.fn().mockReturnValue(mockInteraction);
+
 describe('sessionRoutes', () => {
   const sessionRequest = createRequester({
-    anonymousRoutes: sessionRoutes,
+    sessionRoutes,
     provider: new Provider(''),
     middlewares: [
       async (ctx, next) => {
         ctx.log = jest.fn();
+        ctx.logSession = jest.fn();
```

Comment:
what's this?

Reply:
Removed. (It will be used in the next PR)